zanata-client-commands/src/main/java/org/zanata/client/commands/pull/PullOptionsImpl.java

```diff
@@ -166,6 +167,16 @@ public boolean isContinueAfterError() {
     }
 
     @Override
+    public int getMinDocPercent() {
+        return this.minDocPercent;
+    }
+
+    @Option(name = "--min-doc-percent", aliases = "-m", usage = "When specified, will only pull down translation files that have translated or better statistics above the specified percentage")
```

Comment:
I think `--min-doc-translated-percent` would be clearer, even if it is longer.  Same goes for the Maven option.